cordova-lib/src/plugman/install.js

```diff
@@ -71,7 +71,8 @@ module.exports = function installPlugin(platform, project_dir, id, plugins_dir,
     project_dir = cordovaUtil.convertToRealPathSafe(project_dir);
     plugins_dir = cordovaUtil.convertToRealPathSafe(plugins_dir);
     options = options || {};
-    options.is_top_level = true;
+    if (options.is_top_level === undefined)
```

Comment:
I meant that IMO the purpose of this check is to validate id someone already specified this value, and in this case it's beter to use `hasOwnProperty` which does exactly what it means